.github/workflows/ci.yml.bak

```diff
@@ -0,0 +1,103 @@
+---
+
+name: Packer
```

Comment:
This file should probably just get deleted if it's not used anymore

Reply:
Thanks for catching that